src/views/ModelPlan/ReadOnly/OperationalNeeds/index.tsx

```diff
@@ -51,7 +57,7 @@ const ReadOnlyOperationalNeeds = ({ modelID }: { modelID: string }) => {
         modelID={modelID}
         type="needs"
         readOnly
-        hiddenColumns={['Subtasks']}
+        isExportingPDF={isExportingPDF}
```

Comment:
There is a little prop drilling here that could maybe leverage a context we have for exporting pdf.  You can get the state of the export from this code.  Not huge deal just a suggestion.  I think there should already be the context wrapper - PrintPDFWrapper - around all of readonly

`const { isPrintPDF } = useContext(PrintPDFContext);`

Reply:
Great suggestion! I didn't know we had something like that already existing.